test/Microsoft.AspNetCore.Mvc.Core.Test/DependencyInjection/ApplicationModelConventionExtensionsTest.cs

```diff
@@ -13,6 +13,33 @@ namespace Microsoft.Extensions.DependencyInjection
     public class ApplicationModelConventionExtensionsTest
     {
         [Fact]
+        public void DefaultParameterModelConvention_AppliesToAllParametersInApp()
+        {
+            // Arrange
+            var app = new ApplicationModel();
+            app.Controllers.Add(new ControllerModel(typeof(HelloController).GetTypeInfo(), new List<object>()));
+            app.Controllers.Add(new ControllerModel(typeof(WorldController).GetTypeInfo(), new List<object>()));
+
+            var options = new MvcOptions();
+            options.Conventions.Add(new SimpleParameterConvention());
+
+            // Act
+            options.Conventions[0].Apply(app);
+
+            // Assert
+            foreach (var controller in app.Controllers)
+            {
+                foreach (var action in controller.Actions)
+                {
+                    foreach (var parameter in action.Parameters)
+                    {
+                        Assert.True(parameter.Properties.ContainsKey("TestProperty"));
```

Comment:
Try not to use `Assert.True` if possible. Use `Assert.Contains` instead. `Assert.True` is like the Guy Fieri of assertions